libpod/container.go

```diff
@@ -206,6 +206,10 @@ type ContainerState struct {
 	// and not delegated to the OCI runtime.
 	ExtensionStageHooks map[string][]spec.Hook `json:"extensionStageHooks,omitempty"`
 
+	// NetworkDescriptions describe the relationship between a CNI
+	// network and an interface names
+	NetworkDescriptions ContainerNetworkDescriptions `json:"networkDescriptions,omitempty"`
```

Comment:
Any chance we could rename this to `NetInterfaceDescriptions` or similar? Something that makes it clear this maps network name to interface number.